app/helpers/pretty-bytes.js

```diff
@@ -2,4 +2,12 @@ import { helper } from '@ember/component/helper';
 
 import prettyBytes from 'pretty-bytes';
 
-export default helper(([bytes], options) => prettyBytes(bytes, options));
+/**
+ * See https://github.com/rust-lang/crates.io/discussions/7177
+ */
+export default helper(([bytes], options) =>
+  prettyBytes(bytes, {
+    binary: true,
+    ...options,
+  }),
```

Comment:
@Turbo87, what's the point of "making it possible to override these defaults", if it is never (AFAIU) overridden anywhere?

In https://github.com/rust-lang/crates.io/discussions/7177#discussioncomment-7154109 I intentionally wrote `{...options, binary: true}` so that binary prefixes will be anywhere and everywhere (that was the point from the start).